### PR TITLE
deps: bump gorilla/scheme to v1.4.1

### PR DESCRIPTION
**Pull request by dylanhitt**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,8 +7,9 @@ require (
 	github.com/go-playground/validator/v10 v10.19.0
 	github.com/golang-jwt/jwt/v5 v5.2.1
 	github.com/gomarkdown/markdown v0.0.0-20231222211730-1d6d20845b47
-	github.com/gorilla/schema v1.2.1
+	github.com/gorilla/schema v1.4.1
 	github.com/hashicorp/golang-lru/v2 v2.0.7
+	github.com/rs/cors v1.11.0
```
Comment (EwenQuim) [inline go.mod:12]: I don't like having this in go.mod, it make it looks like Fuego needs this packagez but it's only used in tests

Comment (dylanhitt) [reply go.mod:12]: I’ll take a look. But all I did was bump scheme and go mod tidy. Maybe I fat fingered something.

Comment (dylanhitt) [reply go.mod:12]: Okay just looked at schema. Not sure how this is needed at all.

Comment (dylanhitt) [reply go.mod]: Should be fine now. Thank you for the catch 😄

**Pull request merged**